docs/apis-clients/tasklist-api/enums/sort.mdx

```diff
@@ -0,0 +1,63 @@
+---
+id: sort
+title: Sort
+hide_table_of_contents: false
+---
+
+export const Bullet = () => (
+  <>
+    <span
+      style={{
+        fontWeight: "normal",
+        fontSize: ".5em",
+        color: "var(--ifm-color-secondary-darkest)",
+      }}
+    >
+      &nbsp;●&nbsp;
+    </span>
+  </>
+);
+
+export const SpecifiedBy = (props) => (
+  <>
+    Specification
+    <a
+      className="link"
+      style={{ fontSize: "1.5em", paddingLeft: "4px" }}
+      target="_blank"
+      href={props.url}
+      title={"Specified by " + props.url}
+    >
+      ⎘
+    </a>
+  </>
+);
+
+export const Badge = (props) => (
+  <>
+    <span class={"badge badge--" + props.class}>{props.text}</span>
+  </>
+);
+
+State of the task.
+
+```graphql
+enum Sort {
+  ASC
+  DESC
+}
+```
+
+### Values
+
+#### [<code style={{ fontWeight: 'normal' }}>Sort.<b>ASC</b></code>](#)
+
+> Ascending
+
+#### [<code style={{ fontWeight: 'normal' }}>Sort.<b>DESC</b></code>](#)
+
+> Descending
+
+### Member of
+
+[`TaskQuery`](/docs/apis-clients/tasklist-api/inputs/task-query) <Badge class="secondary" text="input"/>
```

Comment:
```suggestion
[`TaskQuery`](../inputs/task-query.mdx) <Badge class="secondary" text="input"/>
```
re: the links exiting the "next" version and entering the 8.1 version -- 

@christinaausley and I noticed this in [her PR that restructures the apis-clients folder](https://github.com/camunda/camunda-platform-docs/pull/1811/files#diff-4c19156bafac5a18eb22cf7e4640e3d68e14679bd08fba0a42f6e2d647d4072f) -- all of the links within the Tasklist API documentation are referencing the current version of the documentation specifically, because they don't include a `.md` (or `.mdx`) suffix.

Docusaurus interprets those as browser URLs, not paths to documents, and the URLs without a version specified in them emit the current version. 

If we add the `.md`/`.mdx` suffix to the path, Docusaurus will instead interpret it as a path to another document....which will keep the link within the version currently being viewed.

Further, since these URLs are within the same subtree of the documentation, they should be relative from the current document, rather than starting from the root.

We do have [guidance on how to format internal links](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#internal-links), but it is easy to overlook.

## Actions

1. We should correct all the links in this PR to include the file-type suffix, and be relative from the viewed file instead of the root. I consider this the responsibility of the PR authors.
2. I _think_ @christinaausley has accounted for these types of changes to the previously existing Tasklist API docs in #1811. If I'm incorrect, Christina and I can work out how to track the work of correcting the previously existing links.
3. I've created [an issue to automate detection of improperly formatted links](https://github.com/camunda/camunda-platform-docs/issues/1876).
4. If anyone has suggestions to improve the [guidance on formatting internal links](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#internal-links), we can handle those in a separate PR!

Reply:
Hey @pepopowitz, I added the extension to the files, and now the checks are working fine.